pipestat_example/.looper.yaml

```diff
@@ -0,0 +1,9 @@
+pep_config: ./project/project_config_pipestat.yaml # pephub registry path or local path
+output_dir: "./results"
+pipeline_interfaces:
+  sample:  ./pipeline/pipeline_interface.yaml
+pipestat:
+  sample:
+    sample_name: frog_1
```

Comment:
sample_name should *not* be included here in the looper config; the sample name varies by sample, so this makes no sense.